pages/prelude/terra-box/index.md

```diff
@@ -5,24 +5,64 @@ datatable: false
 
 This page describes how the project is organized and explains a few of concepts that you should understand to work with WDP deployments.
 
-## Project structure
+## Terra-Box Definition Tree (TDT)
 
-Terra-box is organized in two main parts:
+As we pointed before, WDP is composed by several microservices or components which work together. On Terra-Box, a set of components is defined by a "Terra-Box Definition Tree" (TDT). It is a way of organizing the Terraform files which defines how a component is deployed on Kubernetes. A TDT is organized on the following folders:
 
-* **Base**: It includes the definition of the main components WDP.
+* modules
+* main
+* template
 
-* **Platforms**: It defines components and configurations which add support to deploy WDP in many cloud providers (AWS, GCP, Azure, ...) or on-premise environments.
+### modules
 
-## Modules for WDP components
+This is the place where the WDP components are defined. Each component definition is created as a Terraform module and it can be imported and used by a higher level module (defined in "main").
 
-## Main modules
+For instance, Wizz-Vis is a WDP component defined in `modules`. It has two Terraform files:
 
-## Templates
+* `main.tf` where the kubernetes resources which deploys Wizz-Vis are defined. This module creates several kubernetes [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment), a [service](https://kubernetes.io/docs/concepts/services-networking/service/) and a [secret](https://kubernetes.io/docs/concepts/configuration/secret).
 
-## Overriding modules
+* `vars.tf` where the module [Input Variables](https://learn.hashicorp.com/terraform/getting-started/variables) (commonly simply called as **vars**) are declared. Module vars let us to parametrize configuration details of the component. E.g.: The password of the admin user.
 
-## ...
+Most of this components can be deployed on kubernetes in a isolated way but some of them might have dependencies.
+
+### main
+
+In order to deploy many WDP components at the same time, two "high level" modules are defined:
+
+* **global** - It includes these components which are considered as part of the WDP core (global components). This components are shared for all organizations and users of that WDP deployment.
+* **user** - It includes all those multi-tenant components which have an instance for each organization or user (user components).
+
+In `main` modules (global and user) we can find a `.tf` file for each component defined in `modules` we want to include in. Each `.tf` file imports the module and configures related input variables.
+
+Although there are several Terraform files in a `main` module, all of them are considered as the same main module.
+
+For example, the `global` module includes some components like kafka, zookeeper or druid. All those components are deployed as part of the same main module.
+
+### templates
```

Comment:
May be we should explain var file priority, indicating that config has priority over flavour and flavour over static